tests/integration/targets/mso_schema_template_contract_filter/tasks/main.yml

```diff
@@ -74,62 +74,62 @@
     template: Template 3
     state: present
 
-- name: Ensure Filter 1 exist
+- name: Ensure Filter1 exist
```

Comment:
Are space not allowed anymore?

Reply:
correct for filter/entry/contract a validation rule is triggered by NDO